store/postgres/namespace_repository.go

```diff
@@ -113,6 +113,9 @@ func (repo *namespaceRepository) Save(ctx context.Context, spec models.Namespace
 	} else if err != nil {
 		return errors.Wrap(err, "unable to find namespace by name")
 	}
+	if len(spec.Config) == 0 {
+		return store.ErrEmptyConfig
+	}
```

Comment:
Does this mean we can't save a namespace with empty configs? I don't think this will be nice right?

Reply:
Whenever the namespace is not present and config is empty, it will still be created. However, if namespace is already there, it will not update on empty configs.